README.md

```diff
@@ -3,14 +3,11 @@
 
 [![Build Status](https://travis-ci.org/codemix/fast.js.svg?branch=master)](https://travis-ci.org/codemix/fast.js)
 
-Faster user-land reimplementations for several common builtin native JavaScript functions.
+## Introduction
+Fast.js is your solution for easily writing very fast JavaScript programs like web servers. This library contains a collection of small changes for several common built-in native JavaScript functions, which make writing fast JavaScript programs easier. These micro-optimizations are capable of making your programs considerably faster.  It includes fast replacements for several built-in native methods such as .forEach, .map, .reduce etc, as well as common utility methods such as .clone.  
```

Comment:
> Fast.js is your solution for easily writing very fast JavaScript programs like web servers.

:-1: - It's not specific to web servers, nor is it a complete solution... It's not even recommended for browser use yet. Also, you lost the formatting on method names in the last line.